test/common.sh

```diff
@@ -50,7 +50,7 @@ if [ -n "$TRACE" ]; then
   set -x
 fi
 
-export KIND_K8S_VERSION="${KIND_K8S_VERSION:-"v1.28.0"}"
+export KIND_K8S_VERSION="${KIND_K8S_VERSION:-"v1.28.3"}"
```

Comment:
It seems like the latest kind node image remains `v.1.28.0`:
https://hub.docker.com/r/kindest/node/tags .

Reply:
Terrific good catcher I will revert this one

Reply:
@Kavinjsir 

Can we merge this one now?
WDYT? Could you LGTM this one?